### PR TITLE
Refactoring + new UI design

### PR DESCRIPTION
**Pull request by LeonidErmak**
split to multiple components the token page

### CODE REVIEW

File: frontend/src/components/elements/PaginationBox/PaginationBox.jsx [hunk 1]
```diff
@@ -1,60 +1,64 @@
...
+    };
+
+    const handlePagePrev = () => {
+        currentPage > 1 ? handlePage(--currentPage) : handlePage(currentPage);
+    };
+
+    return (
+        <div className={ isDarkMode ? styles.rootDark : styles.root }>
+            <Pagination>
+               {/* <Pagination.First onClick={() => handlePage(1)}/>*/}
```
Comment (andrii4ire) [inline frontend/src/components/elements/PaginationBox/PaginationBox.jsx:43]: Remove useless code

File: frontend/src/components/elements/PaginationBox/PaginationBox.jsx [hunk 2]
```diff
@@ -1,60 +1,64 @@
...
+                            onClick={() => handlePage(item)}
+                            active={item === currentPage}
+                        >
+                            {item}
+                        </Pagination.Item>
+                    );
+                })}
+
+                <Pagination.Next onClick={handlePageNext}/>
+                {/*<Pagination.Last onClick={() => handlePage(pages.length)}/>*/}
```
Comment (andrii4ire) [inline frontend/src/components/elements/PaginationBox/PaginationBox.jsx:58]: Remove useless code

File: frontend/src/pages/Tokens/components/OptionsSection/OptionsSection.jsx [hunk 3]
```diff
@@ -0,0 +1,305 @@
...
+                        onClick={() => setState({ expandCreateToken: true })}
+                    />
+                </div>
+            )}
+            <hr/>
+        </div>
+
+    ) : (
+        <div></div>
+        /*<div className="padding-10-0">
```
Comment (andrii4ire) [inline frontend/src/pages/Tokens/components/OptionsSection/OptionsSection.jsx:295]: Remove useless code

Comment (LeonidErmak) [reply frontend/src/pages/Tokens/components/OptionsSection/OptionsSection.jsx]: Done

File: frontend/src/components/elements/PaginationBox/PaginationBox.jsx (hunk 2 above)
Comment (LeonidErmak) [reply frontend/src/components/elements/PaginationBox/PaginationBox.jsx]: Done

File: frontend/src/components/elements/PaginationBox/PaginationBox.jsx (hunk 1 above)
Comment (LeonidErmak) [reply frontend/src/components/elements/PaginationBox/PaginationBox.jsx]: Done

**Pull request merged**